records/config.go

```diff
@@ -97,7 +97,10 @@ func SetConfig(cjson string) (c Config) {
 	}
 
 	// read configuration file
-	usr, _ := user.Current()
+	usr, err := user.Current()
+	if err != nil {
+		logging.Error.Fatalf("user.Current: %v", err)
```

Comment:
Please log a message that doesn't leak implementation details. Something like: `failed to retrieve OS user: %s`